docs/Advanced-Usage/Debugging-in-Software/RTL-Simulation.rst

```diff
@@ -109,40 +108,38 @@ Run all RISCV-tools assembly and benchmark tests on a verilated simulator.
 ::
 
     [in firesim/sim]
-    make DESIGN=FireSimNoNIC
-    make DESIGN=FireSimNoNIC -j run-asm-tests
-    make DESIGN=FireSimNoNIC -j run-bmark-tests
+    make DESIGN=FireSim
```

Comment:
If we only have one DESIGN by default we should just remove the DESIGN=X from these make invocations.

Reply:
The FireSim method of specifiying DESIGN= should be replaced with Configs that set BuildTop IMO, but that can be done in the future.

Reply:
wait, isn't this the case already with this PR? you can give configs as args here too

Reply:
Right, but I mean the way the docs suggest `DESIGN=GCD` needs to be changed as well. Specifying a Top using that method should still work, as it replaces the FireSimHarness with something else.

Ideally, we should be using the same FireSimHarness in all designs, and simply changing the value of the BuildTop key. For these examples,  we need new Config objects which set the BuildTop key to that custom top.

Reply:
got it